### PR TITLE
config_tools: refactor configurator for web page cache issue

### PR DESCRIPTION
**Pull request by Weiyi-Feng**
refactor configurator for web page cache issue

Signed-off-by: Weiyi Feng <weiyix.feng@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by GGGZZZRRR** [comment]
@Weiyi-Feng please add "Tracked on"

File: misc/config_tools/scenario_config/validator.py [hunk 1]
```diff
@@ -101,7 +107,7 @@ def check_semantics(self, board_etree, scenario_etree):
 
     @staticmethod
     def format_paths(unified_node, parent_map, report_on, variables):
-        elems = elementpath.select(unified_node, report_on, variables = variables, parser = elementpath.XPath2Parser)
+        elems = elementpath.select(unified_node, report_on, variables=variables)
```
Comment (junjiemao1) [inline misc/config_tools/scenario_config/validator.py:110]: Why do we want to drop the `parser` optional parameter?

Also I see quite a few stylish changes in this file.

**Review by junjiemao1 — CHANGES_REQUESTED** [review]

File: misc/config_tools/scenario_config/validator.py (hunk 1 above)
Comment (Weiyi-Feng) [reply misc/config_tools/scenario_config/validator.py:110]: Maybe merge issue...Sorry for that..

**Comment by Weiyi-Feng** [comment]
[External_System_ID] ACRN-8729

**Pull request merged**